python/dask_cudf/dask_cudf/groupby.py

```diff
@@ -613,6 +483,25 @@ def groupby_agg(
     return new_dd_object(graph, gb_agg_name, _meta, divisions)
 
 
+@_dask_cudf_nvtx_annotate
+def _make_groupby_agg_call(gb, by, aggs, split_every=None, split_out=1):
```

Comment:
Perhaps we should remove the default values for `split_every` and `split_out` to ensure that the caller always provides those values, rather than the internal defaults. That would have prevented the problem I mentioned in my other comment.
```suggestion
def _make_groupby_agg_call(gb, by, aggs, split_every, split_out):
```